libs/common-types/src/tokens.rs

```diff
@@ -7,13 +7,22 @@ use serde::{Deserialize, Serialize};
 use sp_runtime::format_runtime_string;
 use sp_std::vec::Vec;
 
+#[derive(
+	Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
+)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum PermissionedCurrency {
+	// TODO: Tranche variant from CurrencyId should be moved in here.
+}
+
 #[derive(
 	Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
 )]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum CurrencyId {
 	Native,
 	Usd,
+	Permissioned(PermissionedCurrency),
```

Comment:
Could you move this variant to the end of the list? The variants are encoded as numbers and for some xcm integrations in XTokens (so Native == 0/1, Usd = 1/2, etc) so it will be good to leave the existing variants what they were.

Reply:
Good point!